examples/webhook.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/xanzy/go-gitlab"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+// parse errors
+var (
+	errInvalidHTTPMethod             = errors.New("invalid HTTP Method")
+	errMissingGitLabEventHeader      = errors.New("missing X-Gitlab-Event Header")
+	errGitLabTokenVerificationFailed = errors.New("X-Gitlab-Token validation failed")
+	errEventNotFound                 = errors.New("event not defined to be parsed")
+	errParsingPayload                = errors.New("error parsing payload")
+)
```

Comment:
While this is perfectly fine, I prefer to not declare these up front in this example. In my opinion it takes the focus away from the actual example and they are used only once anyway...